src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java

```diff
@@ -154,18 +154,15 @@ void testDocumentPayloadValues() {
         assertThat(payload.getIdamSurname()).isEqualTo("Barry Allen");
     }
 
-    @ParameterizedTest
-    @EnumSource(value = SetAsideActions.class, names = {"GRANT", "REFUSE"})
-    void givenSetAsideState_andPostHearingsIsEnabled_thenReturnSetAsideDecisionNotice(SetAsideActions setAsideActions) {
```

Comment:
Why is it no longer required to test grant and refused?

Reply:
as it doesn't check the action anymore, it just checks the review type